test/features/steps/steps.py

```diff
@@ -21,3 +20,73 @@ def log_in(context):
         And I press the element with xpath "//button[contains(string(), 'Login')]"
         Then I should see an element with xpath "//a[contains(string(), 'Log out')]"
     """)
+
+
+@step(u'I go to dataset "{name}"')
```

Comment:
if possible, please extract dataset-related steps into a separate class so that it can be shared with other extensions